kalite/contentload/management/commands/channels/khan.py

```diff
@@ -79,39 +90,100 @@ def build_full_cache(items, id_key="id"):
     """
     for item in items:
```

Comment:
Well, all this code should be shifted out of distributed, frankly. We should be building channels on the central server.